client/my-sites/site-settings/manage-connection/disconnect-site-link.jsx

```diff
@@ -55,30 +35,14 @@ class DisconnectSiteLink extends Component {
 				<QuerySitePlans siteId={ siteId } />
```

Comment:
Do we still need this one?

Reply:
Looks like we don't. Good catch, I'll remove it!